pallets/living-assets-ownership/src/lib.rs

```diff
@@ -14,14 +15,22 @@ pub mod pallet {
 	use crate::functions::convert_asset_id_to_owner;
 
 	use super::*;
-	use frame_support::pallet_prelude::{OptionQuery, ValueQuery, *};
+	use frame_support::{
+		pallet_prelude::{OptionQuery, ValueQuery, *},
+		BoundedVec,
+	};
 	use frame_system::pallet_prelude::*;
 	use sp_core::{H160, U256};
 
 	/// Collection id type
-	/// TODO: use 256 bits
 	pub type CollectionId = u64;
 
+	/// Base URI limit type
+	pub type BaseURILimit = ConstU32<255>;
```

Comment:
sgtm

Reply:
> > in case extenal code need to know the bound of BaseURI it can retrive it by [BoundedVec api](https://paritytech.github.io/substrate/master/sp_core/bounded_vec/struct.BoundedVec.html#method.bound)
> 
> hmm, can you suggest any code example? I am curious how this can be achieved by querying an associated type?

https://github.com/freeverseio/laos-ownership-node/pull/110/commits/2396f85d6c7aa545def98a91c310f9068d24af39

Reply:
On the discussion about moving the constant into the Config trait of the pallet I opened this post:
https://substrate.stackexchange.com/questions/9633/best-practices-setting-non-configurable-types-in-a-substrate-pallet

Reply:
as you suggested I moved the BaseURILimit to the trait Config.
I added the documentation to point out that it should be a **fixed value** following the standards.
```
/// Specifies the advised maximum length for a Base URI.
///
/// This adheres to the URI standard (RFC 3986) which dictates a limit 
/// of 2,083 characters for URIs.
///
/// The base should be capped at 2,050 characters in length. This ensures room for 
/// the token URI formation, where it combines `BaseURILimit`, a `'/'`, and a `tokenID` 
/// (which takes up 33 characters).
```